sdk/core/azure-core/inc/azure/core/operation.hpp

```diff
@@ -142,9 +147,9 @@ namespace Azure { namespace Core {
     }
 
     /**
-     * @brief Calls the server to get updated status of the long-running operation.
+     * @brief Gets updated status of the long-running operation.
      *
-     * @param context #Azure::Core::Context allows the user to cancel the long-running operation.
+     * @param context #Azure::Core::Context allows canceling of the long-running operation.
```

Comment:
```suggestion
     * @param context #Azure::Core::Context allows cancelling of the long-running operation.
```
Super nit, just for uniformity; We use LL in other places.